src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,72 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String dataFromFile = getDataFromFile(fromFileName);
+
+        String textToWrite = searchForIdenticalAndWrite(dataFromFile);
+
+        writeDataToFile(textToWrite, toFileName);
+    }
+
+    public String getDataFromFile(String file) {
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(file))) {
+            StringBuilder strBuild = new StringBuilder();
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                strBuild.append(value).append(" ");
+                value = bufferedReader.readLine();
+            }
+
+            return strBuild.toString();
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file");
```

Comment:
The exception message 'Can't read file' could be more descriptive by including the file name. Consider using: `throw new RuntimeException("Can't read file: " + file, e);` to provide more context.